src/app/ipv/middleware.js

```diff
@@ -379,6 +379,27 @@ module.exports = {
     }
   },
 
+  handleDocumentStartPageOptions: async (req, res, next) => {
+    try {
+      if (!req.session?.ipvSessionId) {
+        const err = new Error("req.ipvSessionId is missing");
+        err.status = HTTP_STATUS_CODES.UNAUTHORIZED;
+        logError(req, err);
+
+        req.session.currentPage = "pyi-technical-unrecoverable";
+        return res.redirect(`/ipv/page/pyi-technical-unrecoverable`);
+      }
+      if (req.body?.journey === "next/multiple-doc-check") {
+        await handleJourneyResponse(req, res, "journey/document");
+      } else if (req.body?.journey === "next/postoffice-start") {
```

Comment:
I'm thinking we could simplify this and just handle these with "yes" -> `next` and "no" -> `end` events, which we can handle in the state machine. In fact if you make the radio values "next" and "end" it should be handled by the handleJourneyAction handler so maybe we don't need this custom handler?